practices/smart_goals.md

```diff
@@ -0,0 +1,123 @@
+---
+slug: smart-goals
+stages:
+  - development
+short_description: SMART Goals is a tool that can be used to plan and achieve your goals – both personal and work ones. This tool helps you to make your goals achievable by assessing whether they are Specific, Measurable, Attainable, Relevant, and Time-bound. 
+tags:
+  - SMART Goals
+  - tool
+  - planning
+  - realistic goals
+keywords:
+  - smart
+  - goals
+  - tool
+  - planning
+  - methodology
+---
+
+# SMART Goals
+
+## TL;DR
+
+SMART Goals is a tool that can be used to plan and achieve your goals – both personal and work ones. This tool helps you to make your goals achievable by assessing whether they are **S**pecific, **M**easurable, **A**ttainable, **R**elevant, and **T**ime-bound. 
+
+![](/files/smart3.png) 
+
+## What Are SMART Goals?
+
+**SMART** is an acronym that you can use to guide your goal setting. Using SMART Goals sets boundaries and defines the steps you will need to take, resources necessary to get there, and milestones that indicate progress along the way. To make sure your goals are clear and reachable, each one should be:
+
+- **S**pecific (simple, sensible, significant).
+- **M**easurable (meaningful, motivating).
+- **A**ttainable (agreed, achievable).
+- **R**elevant (reasonable, realistic and resourced, results-based).
+- **T**ime bound (time-based, time limited, time/cost limited, timely, time-sensitive).
+
+### Specific
+
+Specify your goal to be able to focus your efforts. The following questions can help you: 
+
+- What do I want to accomplish?
+- Why is this goal important?
+- Who is involved?
+- Where is it located?
+- Which resources or limits are involved?
+
+### Measurable
+
+With measurable goals set, you can track your progress and stay motivated. Clearly set deadlines will help you to assess your progress. 
+A measurable goal should address questions such as:
+
+- How much?
+- How many?
+- How will I know when it is accomplished?
+
+### Attainable
+
+Your goal also needs to be realistic and attainable to be successful. When you set an achievable goal, you may be able to identify previously overlooked opportunities or resources that can bring you closer to it.
+To set an attainable goal, you will have to deal with the following questions:
+
+- How can I accomplish this goal?
+- How realistic is the goal, based on other constraints, such as financial factors?
+
+**NOTE**: Do not set goals that depend on someone else. Only goals that depend on you, your effort, and your skills are convenient ones. 
```

Comment:
I don't recall DXKB having this way for notes, but I can be easily wrong. However, I would suggest removing the **NOTE** since this information is important for successful implementation.
```suggestion
Do not set goals that depend on someone else. Only goals that depend on you, your effort, and your skills are convenient ones. 
```